main_hot_reload/main_hot_reload.odin

```diff
@@ -2,12 +2,12 @@
 
 package main
 
-import "core:dynlib"
-import "core:fmt"
-import "core:c/libc"
-import "core:os"
-import "core:log"
-import "core:mem"
+import "base:dynlib"
+import "base:fmt"
```

Comment:
Thanks for the PR. However, something is wrong with this. fmt is part of core. The only packages in base are "runtime", "intrinsics" and "builtin". You can see them here: https://github.com/odin-lang/Odin/tree/master/base

Reply:
Fixed